File: src/main/java/io/nozemi/minecraft/commands/TestCommand.java
package io.nozemi.minecraft.commands;

import dev.alangomes.springspigot.context.Context;
import org.bukkit.entity.Player;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Component;
import picocli.CommandLine;

@Component
@CommandLine.Command(name = "heal")
public class TestCommand implements Runnable {

    private final Context context;

    @Autowired
    public TestCommand(Context context) {
        this.context = context;
    }

    @Override
    public void run() {
        Player player = context.getPlayer();
        player.setHealth(20);
    }
}

File: src/main/kotlin/io/nozemi/minecraft/commands/TestCommand.kt
package io.nozemi.minecraft.commands

import dev.alangomes.springspigot.context.Context
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Component
import picocli.CommandLine

@Component
@CommandLine.Command(name = "heal")
class TestCommand @Autowired constructor(private var context: Context) : Runnable {

    override fun run() {
        val player = context.player
        player.health = 20.0
        player.sendMessage("You have been healed!");
    }

}